README.md

```diff
@@ -260,6 +261,36 @@ knockknock matrix \
     sleep 10
 ```
 
+### DingTalk
+DingTalk is now supported thanks to [@wuutiing](https://github.com/wuutiing). Given DingTalk chatroom robot's webhook url and secret/keywords(at least one of them are set when creating a chatroom robot), your notifications will be sent to reach any one in that chatroom.
+
+#### Python
+
+```python
+from knockknock import dingtalk_sender
+
+webhook_url = "<webhook_url_to_your_dingtalk_chatroom_robot>"
+@slack_sender(webhook_url=webhook_url, secret="<your_robot_secret_if_set>", keywords=["<list_of_keywords_if_set>"])
```

Comment:
Small typo: `dingtalk_sender` instead of `slack_sender `

Reply:
> Small typo: `dingtalk_sender` instead of `slack_sender `

sure, fixed